### PR TITLE
comment

### PR DESCRIPTION
**Pull request by Alfarisii**

### CODE REVIEW

**Comment by Alfarisii** [comment]
good

**Comment by muhammadfaiz12** [comment]
Looks good to me !

File: logic.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-def show_all_day(db,min,max):
-    
+def show_all_day(db,min,max)-> int:
+    #min and mix is for filtering all day.
     if min is not None and max is not None:
```
Comment (muhammadfaiz12) [inline logic.py:3]: the if is good

File: logic.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-def show_all_day(db,min,max):
-    
+def show_all_day(db,min,max)-> int:
+    #min and mix is for filtering all day.
```
Comment (Alfarisii) [inline logic.py:2]: ok noted

**Pull request merged**